### PR TITLE
Update white-label scripts

### PR DESCRIPTION
**Pull request by martinjeret**
Question from Slack community said:

In the docs, you recommend to use the sed tool. But I'm not sure of all the places that need changing. It would be great if you can include a section in the docs with specific instructions on how to change bundle/application IDs. 

the answer is here: https://github.com/codemagic-ci-cd/white-label-demo-project so I thought to just add this info to the top of this page so people can self-serve.

### CODE REVIEW

File: content/knowledge-white-label/white-label-scripts.md [hunk 1]
```diff
@@ -213,4 +213,4 @@ In the following example, to trigger builds for clients `001`, `002` and `003` a
 
 The **Codemagic REST API** can also be used for white label solutions where a dashboard is made available to your customers so they can customize an app themselves. This means they could upload their own icons, images, etc. to brand their app and then create a new build of their app. It could also be more advanced and allow customers to add their own distribution certificates, provisioning profiles and API keys.
 
-You can find our more about the Codemagic REST API [here](../rest-api/codemagic-rest-api.md)
\ No newline at end of file
+You can find our more about the Codemagic REST API [here](../rest-api/codemagic-rest-api.md)
```
Comment (TahaTesser) [inline content/knowledge-white-label/white-label-scripts.md:216]:
```suggestion
You can find out more about the Codemagic REST API [here](../rest-api/codemagic-rest-api.md).
```

**Review by TahaTesser — CHANGES_REQUESTED** [review]

File: content/knowledge-white-label/white-label-scripts.md (hunk 1 above)
Comment (martinjeret) [reply content/knowledge-white-label/white-label-scripts.md]: cheers

**Comment by codemagic-mohammed** [comment]
`sed` should do it, but we're not using it for the [Android side](https://github.com/codemagic-ci-cd/white-label-demo-project/blob/ab7582936c003fa7ddab4e8ff55f6972f8683f25/codemagic.yaml#L66), we're using [this package](https://pub.dev/packages/change_app_package_name) to update the package name.
So maybe we can unify the process for Flutter apps (the sample project) and use the suggested [rename package](https://pub.dev/packages/rename) (haven't tried it yet) and mention that you can use sed to update your native iOS bundle Id.

What do you think @nevercode-kevin ?

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**